examples/3_Advanced_Topics/3-3_Database-specific_configuration.py

```diff
@@ -69,3 +69,17 @@
 spec_query = spec_query.with_batch_size(5)
 spec_query
 # -
+
+# ## Specification to Specification links
+
+# This package supports traversing specification to specification links, by default all such links will be followed.
```

Comment:
```suggestion
# The Restricted Substances database allows Specification records to be defined in terms of other Specification records. Since this is a recursive relationship, there is in principle no limit to the complexity of these Specification-to-Specification (spec-to-spec) hierarchies. By default, this package will consider every spec-to-spec hierarchy completely, with no truncation imposed.
```